### PR TITLE
SoC: intel: sof_sdw: Add support for product Ripto

### PR DESCRIPTION
**Pull request by RanderWang**
Ripto is another product based on TGL with the same
audio hardware configuration as Volteer.

### CODE REVIEW

File: sound/soc/intel/boards/sof_sdw.c [hunk 1]
```diff
@@ -123,6 +123,15 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
...
 	},
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Google"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "Ripto"),
+		},
+		.driver_data = (void *)(SOF_SDW_TGL_HDMI | SOF_SDW_PCH_DMIC |
+					SOF_SDW_FOUR_SPK),
+	},
```
Comment (plbossart) [inline sound/soc/intel/boards/sof_sdw.c:134]: nitpick on commit message: same audio hardware configuration as Volteer

Comment (RanderWang) [reply sound/soc/intel/boards/sof_sdw.c:134]: thanks

**Comment by RanderWang** [comment]
update this PR, thanks!

**Comment by plbossart** [comment]
Not sure what's going on with CI but now we have failures on Up2 nocodec. Gah, this is painful that instabilities keep coming randomly. This PR has NOTHING to do with Up2 nocodec.
@xiulipan @fredoh9 @marc-hb @lyakh @lgirdwood FYI.

**Comment by xiulipan** [comment]
@plbossart This issue worth debug, it seems we have some issue with our test case error handling. The case did not show any error, but some background log collection error happen.

**Comment by marc-hb** [comment]
> but now we have failures on Up2 nocodec.

We started paying more attention to the sof-logger (and there's more coming, see https://github.com/thesofproject/sof-test/issues/297 and 298)

The two failures in https://sof-ci.01.org/linuxpr/PR2357/build4293/devicetest/ look basically the same:

```
2020-08-13 01:58:18 UTC [REMOTE_ERROR] sof-logger was already dead  # (new test)
```
At the bottom of the (long) `slogger` tab:
```
error: Invalid filename length or ldc file does not match firmware
```

**Comment by keyonjie** [comment]
SOFCI TEST

**Comment by RanderWang** [comment]
This PR only enable sdw support in sdw machine driver, thereby it doesn't lead to APL failed.

**Pull request merged**